src/components/Filters.tsx

```diff
@@ -0,0 +1,81 @@
+import classNames from "classnames";
+import { ContainerType, Filters as TFilters } from "datasource";
+import React, { useCallback, useEffect, useMemo, useState } from "react";
+
+type Props = {
+  onChange: (filters: TFilters) => any;
```

Comment:
`void` instead of `any` - we're not interested in the return value.
Or better, `=> Promise<void>` so the Filter component can show loading state

Reply:
True, will be fixed at another issue, read https://github.com/argo22com/garbage_map/pull/32#discussion_r395877203